setup.cfg

```diff
@@ -16,11 +16,11 @@ classifiers =
   Intended Audience :: Developers
   License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)
   Programming Language :: Python
-  Programming Language :: Python :: 3.8
```

Comment:
Why dropping support for Python 3.8?

Reply:
I reverted it, was trying to check the CI run time difference without 2 extra runners